index.md

```diff
@@ -4,12 +4,12 @@ layout: overview
 ---
 
 Helix is a Rackspace chartered effort to unify the look, feel, and
-functionality of our interfaces&mdash;one system composed of this Helix design
+functionality of our interfaces &mdash; one system composed of this Helix design
 library for designers and a UI toolkit for developers. Together, we make
 customers happy.
 
-<div class="hxRow">
-  <div class="hxCol-5 hxOffset-1">
+<div class="hxRow center">
```

Comment:
For the Helix grid, we should have a separate container for styling like `.center`. See https://rackerlabs.github.io/helix-ui/components/grid/#styling

Reply:
I'll stay away from any of larger layout changes until I better understand all the grid settings.

Reply:
@nimbinatus 

Just to clarify, are you saying we should create a utility class for centering content?  Just making sure you're not implying the helix grid already has the utility class.

Reply:
@araiford, the Helix grid (at least as the documentation defines stuff) require a separate container for styling the contents of a column. 
>**Avoid styling rows and columns.** Add a container within a column to apply styling. 
>
>Because styles for rows and columns are meticulously defined to support the grid definition, any modification to the margin, padding, or geometry of these elements will affect the layout.
>
>- Wrap your content in a `<div>` and style using a custom CSS class.

I don't think the grid has the utility class.

Reply:
ahh -- yeah, that makes total sense.

Reply:
I'm going tbh that I can't figure out how to align the cards with the helix grid without reworking the card content. My solution to get this PR moved forward is to remove the center class and associated css. This will make the content flow back into the former position. Perhaps someone with a better understanding of the grid can pick up that card instead.